.github/workflows/github_actions/actions/github_add_or_modify_file_action.rb

```diff
@@ -0,0 +1,49 @@
+class GithubAddOrModifyFileAction
```

Comment:
I know there's not much plumbing there just yet, but should still make sure to make it a subclass of `Sublayer::Actions::Base` as we'll be adding logging and other things to the base class over time and would want these to all work similarly